index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio project</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-container">
+        <ul class="nav-bar">
+          <li>Welcome</li>
+          <li class="menu">
+            <a href="#"><img src="images/Icon-Menu.svg" alt="Menu" /></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="intro">
+          <h1>Hey there. I'm Khusniddin</h1>
+          <h2>I'm a software developer</h2>
+        </div>
+        <div class="text">
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <ul class="social-list">
+            <li>
+              <a href="#" target="_blank"
+                ><img src="/images/twitter.svg" alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#" target="_blank"
+                ><img src="/images/Linkedinicon.svg" alt="Linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#" target="_blank"
+                ><img src="/images/medium.svg" alt="Medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#" target="_blank"
+                ><img src="/images/github.svg" alt="GitHub"
+              /></a>
+            </li>
+            <li>
+              <a href="#" target="_blank"
+                ><img src="/images/angellist.svg" alt="Angel list"
+              /></a>
+            </li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly make sure your icons are visible on your page, as shown in the picture below the is no icon that is visible. I can suggest you use `png` format for your images instead of using `svg `. because is easier to import and use or you can also import the library for icons it is also better cause you can be able to give any style to your icons :grinning: :grinning: :grinning: 

 ## Your design

![image](https://user-images.githubusercontent.com/95297251/191786350-50b5310c-6c40-4159-a293-2fc3aef98945.png)

##  Figma design

![image](https://user-images.githubusercontent.com/95297251/191791031-8c76e371-ef82-442a-8eb6-13f1ddcb540d.png)